orchagent/main.cpp

```diff
@@ -101,6 +101,7 @@ void usage()
     cout << "    -f swss_rec_filename: swss record log filename(default 'swss.rec')" << endl;
     cout << "    -j sairedis_rec_filename: sairedis record log filename(default sairedis.rec)" << endl;
     cout << "    -k max bulk size in bulk mode (default 1000)" << endl;
+    cout << "    -q zmq_server_address: ZMQ server address (default tcp://127.0.0.1:8020)" << endl;
```

Comment:
>tcp://127.0.0.1:8020 [](http://example.com/codeflow?start=68&length=20)

You can construct the default string by code.

Reply:
This string removed by https://github.com/sonic-net/sonic-swss/pull/2828